multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1771,6 +1875,8 @@ def predict(
         elif self._pipeline == OCR_TEXT_RECOGNITION:
             ret_type = [TEXT, SCORE]
 
+        pl.seed_everything(seed, workers=True)
```

Comment:
Why do we need seeding in prediction?

Reply:
I think it’s a good practice to seed before running models, even in the inference script. This avoids potential unreproducibility issues. An example is https://github.com/NVIDIA/FasterTransformer/blob/bc077a9078b99244c69cfe0dc44af86fd974bc71/examples/pytorch/bert/run_glue.py#L290

Reply:
Specifically T-Few selects a template randomly (both for training and inference as per default in the paper), so not seeding before running it results in reproducibility issues.

Reply:
Here we only seed in `.predict()`. It can't be used by other APIs, e.g., `.predict_proba()`.

Reply:
Ah right, good point!